howtoplay.html

```diff
@@ -16,7 +16,8 @@
 </head>
 
 <body>
-  <div class="whitespace">
+    <a onclick="javascript: document.location='index.html'" id="Close-Button">&#10006;</a>

```

Comment:
It's inline js, can we seperate it?

Reply:
we can separate it but there is no need for this because it is just a hyperlink.

Reply:
You can also use href="index.html" here.